src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -543,7 +543,7 @@ public function formAction($id, Request $request)
         }
 
         $doctrine = $this->getDoctrine()->getManager();
-        $attributeGroups = $doctrine->getRepository('PrestaShopBundle:Attribute')->findByLangAndShop(1, 1);
+        $attributeGroups = $doctrine->getRepository('PrestaShopBundle:Attribute')->findByLangAndShop((int)$languages[0]['id_lang'], (int)['shop_id']);
```

Comment:
Should be:
`$attributeGroups = $doctrine->getRepository('PrestaShopBundle:Attribute')->findByLangAndShop((int) $languages[0]['id_lang'], (int) ['shop_id']);
`